libraries/botbuilder-core/src/turnContext.ts

```diff
@@ -114,11 +122,7 @@ export class TurnContext {
         var mentions = TurnContext.getMentions(activity);
         for (var i=0; i<mentions.length; i++) {
             if (mentions[i].mentioned.id === id) {
-                var mentionNameMatch = mentions[i].text.match(/(?<=<at.*>)(.*?)(?=<\/at>)/i);
-                if (mentionNameMatch.length > 0) {
-                    activity.text = activity.text.replace(mentionNameMatch[0], '');
-                    activity.text = activity.text.replace(/<at><\/at>/g, '');
-                }
+                activity.text = activity.text.replace(mentions[i].text, '').trim();
```

Comment:
I.... don't understand how this works. In C# you've still got a Regex in there. 

Here, I'm probably missing something obvious...

Reply:
I removed RegEx from the Javascript version because stevengum and Scott said that pattern isn't supported on all browsers.  So I am just grabbing everything between the XML tags.  Rather brute force but should work.

Reply:
yep, this should be fine if we can't use regex